ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -1078,6 +1078,8 @@ management_interfaces:
     ipv6_enable: < true | false >
     ipv6_address: < IPv6_address/Mask >
     type: < oob | inband | default -> oob >
+    gateway: < IPv4 address of default gateway in management VRF >
```

Comment:
Should we mention that this is only used for documentation?

Reply:
Thanks,
Adding a comment to state it is for documentation purpose only.